### PR TITLE
[NA]feat(header): Implement header UX new specs

### PR DESCRIPTION
**Pull request by jmarquez-stratio**
### Documentation
- [x] Add the issue in PR description
- [x] Have changelog updated
- [x] You fill the milestone value
- [x] You add some label
- [ ] Have example running in demo app
- [ ] Review id on demo is the same of egeo folder

### Code
- [x] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [x] Have at least 70% tests coverage

### CODE REVIEW

File: projects/egeo/src/lib/st-header/st-header.component.ts [hunk 1]
```diff
@@ -62,10 +62,11 @@ export class StHeaderComponent implements AfterViewInit {
 
    /** @Input {StHeaderMenuOption[]} [menu] Array with menu option to show */
    @Input() menu: StHeaderMenuOption[] = [];
-   /** @Input {StHeaderMenuOption[]} [menu] Array with menu option to show */
+   /** @Input {navigateByDefault} [boolean] True if we want menu to manage navigation, false if navigation will be managed from the outside */
```
Comment (jpascual-stratio) [inline projects/egeo/src/lib/st-header/st-header.component.ts]: change to {boolean} [navigateByDefault]

Comment (jmarquez-stratio) [reply projects/egeo/src/lib/st-header/st-header.component.ts]: changed

File: projects/egeo/src/lib/st-header/README.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+| navigateByDefault | Boolean              | False | True if we want menu to manage navigation, false if navigation will be managed from the outside |         |
 
 ## Outputs
 
-| Property   | Type   | Description                      |
-| ---------- | ------ | -------------------------------- |
-| selectMenu | String | Notify any menu option selection |
+| Property   | Type                        | Description                      |
+| ---------- | --------------------------- | -------------------------------- |
+| selectMenu | StHeaderSelection \| string | Notify any menu option selection |
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-header/README.md]: Remove string

Comment (jmarquez-stratio) [reply projects/egeo/src/lib/st-header/README.md]: done

**Pull request merged**